### PR TITLE
34 hiroshima

### PR DESCRIPTION
**Pull request by kentoyamada100**

### CODE REVIEW

File: analyses/34_hiroshima/index.Rmd [hunk 1]
```diff
@@ -293,7 +293,7 @@ ggplot() +
   + 旧市区町村単位の人口（総人口-外国人人口）は、合併後の市区町村の人口（総人口-外国人人口）と旧市区町村単位の日本人人口を元に推計
 * 郡の分割は、現行の区割りにて分割されている場合のみ許容
 * 離島は、航路で結ばれている本土の市区町村と同一の選挙区にする
-* 市区町村分割を許容しない案、最大市区町村の分割を許容する案をそれぞれ`r format(nsims, big.mark = ",")`作成 
+* 逐次モンテカルロ法(SMC)による2つの独立したシミュレーションにより、市区町村分割を許容しない案、最大市区町村の分割を許容する案をそれぞれ100,000作成
```
Comment (shomiyazaki) [inline analyses/34_hiroshima/index.Rmd:296]: `r format(nsims * 2, big.mark = ",")`

File: analyses/34_hiroshima/index.Rmd [hunk 2]
```diff
@@ -231,9 +231,9 @@ redist::redist.plot.plans(sim_smc_pref_1_sample, optimal_1, pref_map_1) +
   
   geom_sf(data = cities, size = 2, shape = 21, fill = "red") +
   geom_sf_text(data = cities, aes(label = names), size = 3,
-              nudge_x = c(0, 0, 10000), # adjust the position of the labels
-              nudge_y = c(8000, 7000, 2000), # adjust the position of the labels
-              color = c("white", "white", "white"),
+              nudge_x = c(0, 10000, 10000), # adjust the position of the labels
```
Comment (shomiyazaki) [inline analyses/34_hiroshima/index.Rmd:234]: Would you switch 
dark blue and yellow
for a visual reason
Screen Shot 2022-06-06 at 0 24 47
?

**Review by shomiyazaki — CHANGES_REQUESTED** [review]
Hi @kentoyamada100 Thanks for your PR. Please refer to two comments below

**Comment by kentoyamada100** [comment]
@shomiyazaki ok, I've made those changes. Please merge this into main, if it's ok with you

**Pull request merged**